department-of-veterans-affairs/caseflow
# Make /queue/<css_id>/assign default url for judge assign page

Resolves #13913

### Description
`css_id` replaces `user_id` in the `queue/user/assign` url to make the URL more readable for users

### Acceptance Criteria
- [x] When a judge navigates to their assign page from their dropdown, the url uses their css_id
 - [ ] When a judge navigates to their assign page using their user id, the url is updated to use their css_id
- [x] When clicking around the judge assign queue, the URL should not change to use the users id

### Testing Plan
1. Sign in as `BVAAABSHIRE` and navigate to `http://localhost:3000/queue`
2. Ensure the assign page dropdown navigated to the assign page via the user's `css_id`
3. Navigate to the assign page with the user's `user_id` and endure the url properly updates to use the `css_id`: `http://localhost:3000/queue/BVAAABSHIRE/assign`
4. Ensure all of the `Assign Cases` links navigate to the correct url using the user's `css_id`

### User facing changes

#### Updated url in review page dropdown
#### Before
<img width="1439" alt="Screen Shot 2020-04-22 at 4 28 47 PM" src="https://user-images.githubusercontent.com/18618189/80032782-a9c07500-84b9-11ea-99da-019a312799de.png">

#### After
<img width="1433" alt="Screen Shot 2020-04-22 at 2 56 47 PM" src="https://user-images.githubusercontent.com/18618189/80032939-eab88980-84b9-11ea-9452-bd257cb974f3.png">

#### Updated url on assign page
#### Before
<img width="1439" alt="Screen Shot 2020-04-22 at 4 29 54 PM" src="https://user-images.githubusercontent.com/18618189/80033288-716d6680-84ba-11ea-939d-e40406d13170.png">

#### After
<img width="1439" alt="Screen Shot 2020-04-22 at 2 58 31 PM" src="https://user-images.githubusercontent.com/18618189/80033308-7a5e3800-84ba-11ea-80ed-fa5d7b2874a7.png">

#### Assign page links go to correct urls
#### Before
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/18618189/80035603-4edd4c80-84be-11ea-9c13-f514c9f2b9c4.gif)

#### After
![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/18618189/80036442-ba73e980-84bf-11ea-85e5-b2d7b7c3fee5.gif)
